File: app/src/main/java/com/karthik/splash/Models/PhotoDetail/CategoriesItem.java
package com.karthik.splash.Models.PhotoDetail;


import com.google.gson.annotations.SerializedName;


public class CategoriesItem{

	@SerializedName("photo_count")
	private int photoCount;

	@SerializedName("links")
	private Links links;

	@SerializedName("id")
	private int id;

	@SerializedName("title")
	private String title;
}

File: app/src/main/java/com/karthik/splash/models/photodetail/CategoriesItem.kt
package com.karthik.splash.models.photodetail


import com.google.gson.annotations.SerializedName


data class CategoriesItem(
        @SerializedName("photocount") private val photoCount: Int = 0,
        @SerializedName("links") private val links: Links? = null,
        @SerializedName("id") private val id: Int = 0,
        @SerializedName("title") private val title: String? = null
)